### PR TITLE
Add support for iOS/Android's "Add To Homescreen".

### PR DESCRIPTION
**Pull request by russjr08**
@ajacksified Sorry for the trouble, apparently I'm still learning when it comes to the "advanced" stuff of Git.

[This is from the previous pull request](https://github.com/reddit/reddit-mobile/pull/134), but with just one commit, and no mess :+1: 

Thanks for the help by the way!

### CODE REVIEW

File: src/views/layouts/DefaultLayout.jsx [hunk 1]
```diff
@@ -52,10 +52,12 @@ class DefaultLayout extends React.Component {
           <title>{ this.props.title }</title>
           <link href={ baseCSS } rel='stylesheet' />
           <link href={ fancyCSS } rel='stylesheet' media='screen' />
+          <link rel="manifest" href="manifest.json" />
```
Comment (ajacksified) [inline src/views/layouts/DefaultLayout.jsx]: Single quotes here and below

File: src/serverRoutes.es6.js [hunk 2]
```diff
@@ -27,6 +27,55 @@ Disallow: /u/*
 Allow: /
     `;
   });
+
+  router.get('/manifest.json', function * () {
+      this.body = {
```
Comment (ajacksified) [inline src/serverRoutes.es6.js]: Reduce indentation, here and underneath `icons`

**Comment by ajacksified** [comment]
Just noticed a few little style issues, then we're good to merge. I'll try to get this deployed soon after the merge.

**Comment by russjr08** [comment]
@ajacksified Would you also like single quotes in the manifest's JSON data, or keep the double quotes?

**Comment by ajacksified** [comment]
Oh, singles please.

**Comment by russjr08** [comment]
@ajacksified Done :)

**Comment by ajacksified** [comment]
Fantastic. Thanks!

**Comment by ajacksified** [comment]
It's live!

**Comment by russjr08** [comment]
Wow that was quick, thanks a bunch!

**Pull request merged**